project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.13
+sbt.version=0.13.16-M1
```

Comment:
This is necessary for cross-building to work. It adds all the infrastructure for it, including `^` and `^^`.